Client/my-story-illustrator-client/src/my_stories/make_buttons_boring.css

```diff
@@ -0,0 +1,11 @@
+button {
```

Comment:
Repeat of earlier comment, let's get rid of CSS and use chakra ui styles wherever we can.